purplesmurf1998/Autotracks
# Sprint 6 comments

# Description
This PR lets users create/add comments to a vehicle and delete their comments from vehicle.
# How to test

1. Select a vehicle from an inventory
2. Navigate to the bottom right and click on "Write a comment"
3. Write your comment and format it the way you want to
4. Click "Post" to publish your comment
5. To delete the comment, simple press on the trash icon and confirm your request.
# Screenshots
![image](https://user-images.githubusercontent.com/44242944/151051733-c2639751-d6ba-4bea-be6c-4824cdf33ba0.png)
![image](https://user-images.githubusercontent.com/44242944/151051781-4361e65f-8940-4688-8b30-3c35c8061da2.png)
![image](https://user-images.githubusercontent.com/44242944/151051877-d338ace4-59ff-49f8-8994-fe9d526b96ef.png)
![image](https://user-images.githubusercontent.com/44242944/151051905-e3774d8c-758a-433a-8aee-800ec8bbaebe.png)
# Jira Story
[[AP-20] As a user, I would like to comment on vehicles in the dealership](https://soen390-team12.atlassian.net/browse/AP-20?atlOrigin=eyJpIjoiMmRhNTI5NmM5ZDBkNDFjN2IxYzdiNzg2Y2UwOThmYmEiLCJwIjoiaiJ9)
